libs/eo/landing-page/shell/src/lib/layout/eo-landing-page-video-layout.component.ts

```diff
@@ -0,0 +1,134 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  ChangeDetectionStrategy,
+  Component,
+  NgModule,
+  ViewEncapsulation,
+} from '@angular/core';
+import {
+  EoLandingPageColumnLayoutScam,
+  layoutTypeEnum,
+} from './eo-landing-page-column-layout.component';
+
+const selector = 'eo-landing-page-video-layout';
+
+@Component({
+  selector,
+  styles: [
+    `
+      @use '@energinet-datahub/watt/utils' as watt;
+
+      ${selector} {
+        display: block;
+        position: relative;
+      }
+
+      .${selector}__p {
+        @include watt.typography-watt-text-m; // This overrides the styles applied from Angular Material on p tags
+      }
+
+      .${selector}__h2.${selector}__h2 {
+        @include watt.typography-watt-headline-2; // This overrides the styles applied from Angular Material on h2 tags
+        text-transform: none; // This overrides the uppercased styling from watt
+      }
+
+      .${selector}__link.${selector}__link {
+        display: inline-block;
+        color: var(
+          --watt-color-primary
+        ); // This overrides the '--watt-color-primary-dark' color which is currently added by the watt-text-s class
+      }
+
+      .${selector}__development-notification {
+        background: rgba(223, 235, 218, 0.9);
+        top: calc(-10 * var(--watt-space-xs));
+        height: calc(20 * var(--watt-space-xs));
+        width: 100%;
```

Comment:
suggestion: Remove `width` as we usually use block elements as layout elements, for example `<div>`.
```suggestion
```